guides/common/modules/con_performance-tuning-quickstart.adoc

```diff
@@ -26,5 +26,6 @@ The default value is `default`.
 ----
 # {foreman-installer} --tuning "_My_Installation_Size_"
 ----
+. Optional: For some additional checks see xref:Applying_configurations_{context}[].
```

Comment:
Please reword this. To me, it's unclear why/in which case I'd want to follow the link.

Reply:
Done, thank you.